src/client/client.rs

```diff
@@ -1022,6 +1022,23 @@ impl Builder {
         self
     }
 
+    /// Set whether HTTP/1 connections should try to use vectored writes,
+    /// or always flatten into a single buffer.
+    ///
+    /// Note that setting this to false may mean more copies of body data,
+    /// but may also improve performance when an IO transport doesn't
+    /// support vectored writes well, such as most TLS implementations.
+    ///
+    /// Setting this to true will force hyper to use queued strategy
+    /// which may eliminate unnecessary cloning on some TLS backends
+    ///
+    /// Default is `auto`. In this mode hyper will try to guess which
+    /// mode to use
+    pub fn http1_writev(&mut self, enabled: bool) -> &mut Builder {
```

Comment:
I don't have a strong feeling one way or the other, just wanted to hear what you think: should it be possible for a user to set it back to the default? Such as if another library wrapping hyper also wants to expose this option, should they be able to start with a default that doesn't overwrite the default? Or should the manage that themselves, and only call this function if configured to do so?

Reply:
I guess I can make `enabled` be a `impl Into<Option<bool>>`, but that feels like slightly overkill to me.

Reply:
It feels overkill to me too. I thought it, but didn't want to request it. Just felt it was worth talking about 😅

Reply:
I don't mind either way, if you think just a bool is not good enough, I can make the change.